src/nifreeze/cli/run.py

```diff
@@ -67,16 +48,11 @@ def main(argv=None) -> None:
         **extra_kwargs,
     )
 
-    model_kwargs = {}
-
-    if args.ignore_b0:
-        model_kwargs["ignore_bzero"] = True
-
     prev_model: Estimator | None = None
     for _model in args.models:
-        single_fit = _model.lower().startswith("single")
+        single_fit = estimator_kwargs[_model]["single_fit"]
         estimator: Estimator = Estimator(
-            _model.lower().replace("single", ""),
+            _model,
             prev=prev_model,
             single_fit=single_fit,
             model_kwargs=model_kwargs,
```

Comment:
The estimator accepts other keyword arguments:
https://github.com/nipreps/nifreeze/blob/94f6d2708d15fbe85fb850094e956b63d9c047b9/src/nifreeze/estimator.py#L84

To be fixed in a separate PR.